doc/man/hirte-agent.1.md

```diff
@@ -30,6 +30,12 @@ The host used by `hirte-agent` to connect to `hirte`. Must be a valid IPv4 or IP
 
 The port on which `hirte` is listening for connection request and the `hirte-agent` is connecting to. This option will overwrite the port defined in the configuration file.
 
+#### **--address**, **-a**
+
+SD Bus address used by `hirte-agent` to connect to `hirte`. See `man sd_bus_set_address` for its format.
```

Comment:
Just call it "dbus address". This is a standard format used by all dbus libraries, specified in the dbus spec.

Reply:
reference: 
https://dbus.freedesktop.org/doc/dbus-specification.html#addresses

Reply:
Fixed